### PR TITLE
Add support for zwave-js v10

### PR DESCRIPTION
**Pull request by raman325**
Changelog: https://github.com/zwave-js/node-zwave-js/releases/tag/v10.0.0-beta.8

TODO:
- figure out s2 inclusion callbacks with change to options

### CODE REVIEW

File: src/lib/node/message_handler.ts [hunk 1]
```diff
@@ -211,6 +211,9 @@ export class NodeMessageHandler {
       case NodeCommand.waitForWakeup:
         await node.waitForWakeup();
         return {};
+      case NodeCommand.interview:
+        node.interview();
```
Comment (balloob) [inline src/lib/node/message_handler.ts:215]: should this be awaited?

**Review by balloob — APPROVED** [review]

File: src/lib/node/message_handler.ts (hunk 1 above)
Comment (raman325) [reply src/lib/node/message_handler.ts:215]: The docs suggest not waiting for the interview to complete because of how long it can take

Comment (balloob) [reply src/lib/node/message_handler.ts:215]: I think that we should await it. It's up to the caller (Home Assistant) to decide if they want to await the task or not.

Comment (raman325) [reply src/lib/node/message_handler.ts:215]: should we do this in other places too? right now we don't await `node.interviewCC` for the same reason

Comment (balloob) [reply src/lib/node/message_handler.ts:215]: I think so.

File: src/lib/controller/message_handler.ts [hunk 2]
```diff
@@ -100,9 +100,15 @@ export class ControllerMessageHandler {
...
+            ? await driver.controller.beginExclusion(message.unprovision)
+            : await driver.controller.beginExclusion(
+                message.strategy !== undefined
+                  ? { strategy: message.strategy }
+                  : undefined
+              );
+
         return { success };
       }
       case ControllerCommand.stopExclusion: {
```
Comment (MartinHjelmare) [inline src/lib/controller/message_handler.ts:114]: I'm missing the changes needed for this point in the migration guide:

> - The Controller.replaceFailedNode method overload accepting the node ID as the second parameter has been removed. Use the overload with the ReplaceNodeOptions object instead. NOTE: If you do not pass this object, the new node will be included without security.

Comment (raman325) [reply src/lib/controller/message_handler.ts:114]: this was a deprecated signature that was removed. We switched to the new method a while ago, so there's no more that needs to be done here as far as I am aware.

**Pull request merged**